workflows/quickstart/index.js

```diff
@@ -38,9 +38,9 @@ const client = new ExecutionsClient();
  * @param {string} projectId The Google Cloud Project containing the workflow
  * @param {string} location The workflow location
  * @param {string} workflow The workflow name
- * @param {string} searchTerm Optional search term to pass as runtime argument to Workflow
+ * @param {string} runtimeArgs Runtime arguments to pass to the Workflow as JSON string
```

Comment:
It's still optional. We should be assigning a default if no `runtimeArgs` is passed.

Reply:
searchTerm is defined as `null` on line 22 

`const searchTerm = process.argv[5] || null;`

It isn't optional now but the logic captured in the region tag will replace the default `null` with `'{}'` before calling the function

Reply:
ah ha I think i got it, let me know!